src/app/grid/grid-action-strip/grid-action-strip-sample.html

```diff
@@ -1,46 +1,43 @@
 <div class="grid__wrapper">
-    <igx-grid igxPreventDocumentScroll #gridRowEditTransaction [batchEditing]="true" [data]="data" width="100%"
-        height="500px" [rowEditable]="true" [allowFiltering]="true" (onDataPreLoad)="actionstrip.hide()">
+    <igx-grid igxPreventDocumentScroll #grid [data]="data" [primaryKey]="'ProductID'" width="100%"
+        height="500px" [rowEditable]="true" [batchEditing]="true" [allowFiltering]="true">
```

Comment:
When onDataPreload event is fired, do we see still the action strip now? Not sure if this needed to be removed (onDataPreLoad)="actionstrip.hide()"

Reply:
**onDataPreLoad** is the old name for this event, currently, it is **dataPreLoad**, meaning that this event wasn't fired when scrolling up or down and wasn't affecting the action strip in any way. When changing it to its correct name, i.e., dataPreLoad, there wasn't any change in the action strip behavior as it hides when scrolling with or without the event.